src/ui/zcl_abapgit_password_dialog.clas.abap

```diff
@@ -25,6 +25,17 @@ CLASS zcl_abapgit_password_dialog IMPLEMENTATION.
   METHOD popup.
 
     DATA: lx_error TYPE REF TO cx_sy_dyn_call_illegal_form.
+    DATA lv_default_git_uname TYPE string.
+    DATA lv_default_git_email TYPE string.
+
+    lv_default_git_uname = zcl_abapgit_persist_factory=>get_settings( )->read( )->get_default_git_uname( ).
+    lv_default_git_email = zcl_abapgit_persist_factory=>get_settings( )->read( )->get_default_git_email( ).
+
+    IF lv_default_git_uname IS NOT INITIAL.
+      cv_user = lv_default_git_uname.
+    ELSEIF lv_default_git_email IS NOT INITIAL.
+      cv_user = lv_default_git_email.
```

Comment:
this looks suspicious, both uname and email goes into `cv_user` ?

Reply:
perhaps just leave the password popup as it, and have this PR surface the configuration settings only? there are various other exits for the authentication(which might be different than git users)

Reply:
Looks to me that we just default the email address as user name if not supplied. Sounds sensible to me.

Reply:
will the default override what the user has entered as username?

Reply:
No, see line 34 just before your quoted lines
`IF lv_default_git_uname IS NOT INITIAL.`

Reply:
yea, but that default then overrides the repo login? plus affects proxy authentication? I suggest leaving this PR to just surfacing the configuration in the settings page, and leaving the popup as is

![image](https://github.com/abapGit/abapGit/assets/5888506/2ea4b618-37de-4502-8c7d-7681b46bf685)

Reply:
Mmh, ok I get what you mean now. Agree the login is/should be completely separate to the name & email that is added to a commit.

Reply:
Isn't the point of the default to pre-fill the password popup? 

It should happen only if no user has been given yet. Default user having priority over default email is fine for me. 

```abap
  IF cv_user IS INITIAL.
    lv_default_git_uname = zcl_abapgit_persist_factory=>get_settings( )->read( )->get_default_git_uname( ).
    lv_default_git_email = zcl_abapgit_persist_factory=>get_settings( )->read( )->get_default_git_email( ).

    IF lv_default_git_uname IS NOT INITIAL.
      cv_user = lv_default_git_uname.
    ELSEIF lv_default_git_email IS NOT INITIAL.
      cv_user = lv_default_git_email.
    ENDIF.
  ENDIF.
```

Reply:
git is git, the popup is authentication, which already have settings and functionality

it might be okay to change the defaults, but I'd still suggest that for a different PR, so we can properly focus on reviewing it, the authentication is central functionality. Surfacing the settings is low criticality, that should be easy to merge

Reply:
Hi @larshp,

    The changes have been reverted, as per your request. Let me know in case you want me to create a new PR to pre-fill the password popup. 

Best regards,

Francisco Milan.